docs/resources/scoped_token.md

```diff
@@ -0,0 +1,127 @@
+---
+subcategory: "Security"
+---
+# Artifactory Scoped Token Resource
+
+Provides an Artifactory Scoped Token resource. This can be used to create and manage Artifactory Scoped Tokens.
+
+**Note:** Scoped Tokens will be stored in the raw state as plain-text. [Read more about sensitive data in
+state](https://www.terraform.io/docs/state/sensitive-data.html).
+
+
+## Example Usages
+### Create a new Artifactory scoped token for an existing user
+
+```hcl
+resource "artifactory_scoped_token" "scoped_token" {
+  username = "existing-user"
+}
+```
+
+**Note:** This assumes that the user `existing-user` has already been created in Artifactory by different means, i.e. manually or in a separate terraform apply.
+
+### Create a new Artifactory user and scoped token
+```hcl
+resource "artifactory_user" "new_user" {
+  name   = "new_user"
+  email  = "new_user@somewhere.com"
+  groups = ["readers"]
+}
+
+resource "artifactory_scoped_token" "scoped_token_user" {
+  username = artifactory_user.new_user.name
+}
+```
+
+### Creates a new token for groups
+```hcl
+resource "artifactory_scoped_token" "scoped_token_group" {
+  scopes = ["applied-permissions/groups:readers"]
```

Comment:
you need to doc possible values

Reply:
It's in the description, which is taken from API doc.